### PR TITLE
Updated SettingsCogTest to be compatible with Enzyme 3

### PR DESCRIPTION
**Pull request by jmkulwik**
This is part of the effort to upgrade all tests to be compatible with Enzyme 3.
https://codedotorg.atlassian.net/browse/STAR-185

### CODE REVIEW

File: apps/test/unit/lib/ui/SettingsCogTest.js [hunk 1]
```diff
@@ -157,19 +152,14 @@ describe('SettingsCog', () => {
         expect(wrapper).to.be.blank;
       });
 
-      it('calls handleToggleMaker when clicked', () => {
+      it('asks for confirmation when clicked', () => {
```
Comment (jmkulwik) [inline apps/test/unit/lib/ui/SettingsCogTest.js:150]: This one wasn't testing anything in the react component, so I changed it to be a more useful test.

File: apps/test/unit/lib/ui/SettingsCogTest.js [hunk 2]
```diff
@@ -99,29 +106,17 @@ describe('SettingsCog', () => {
         assets.showAssetManager.restore();
       });
 
-      it('is the first menu item', () => {
```
Comment (epeach) [inline apps/test/unit/lib/ui/SettingsCogTest.js:102]: Why was this removed?

File: apps/test/unit/lib/ui/SettingsCogTest.js [hunk 3]
```diff
@@ -68,26 +74,27 @@ describe('SettingsCog', () => {
...
-  it('only shows manage assets when maker toggle is false', () => {
+  it('does not show maker toggle when "showMakerToggle" is false', () => {
     const wrapper = mount(<SettingsCog showMakerToggle={false} />);
-    const cog = wrapper.find(FontAwesome).first();
-    cog.simulate('click');
-    const menuWrapper = getPortalContent(wrapper);
-    const numMenuItems = menuWrapper.find(PopUpMenu.Item).length;
-    expect(numMenuItems).to.equal(1);
+    expect(wrapper.find(ToggleMaker)).to.have.lengthOf(0);
+    // wrapper.instance().open();
```
Comment (epeach) [inline apps/test/unit/lib/ui/SettingsCogTest.js]: Is there a reason to keep these comments?

File: apps/test/unit/lib/ui/SettingsCogTest.js (hunk 2 above)
Comment (jmkulwik) [reply apps/test/unit/lib/ui/SettingsCogTest.js:102]: I was having a really hard time transitioning it to Enzyme 3. Then I started thinking that perhaps there is a reason there is no easy way to unit test this particular thing. I thought about the test itself and it seemed like a bit of an odd thing to unit test, since it was a visual property that doesn't depend on state or logic within the component. The equivalent that I could think of was testing whether a button's color is red when the button never changes. Perhaps it's something that could be covered in an eyes test, but didn't seem like a unit test. 

I'd be interested in hearing your thoughts :)

File: apps/test/unit/lib/ui/SettingsCogTest.js (hunk 3 above)
Comment (jmkulwik) [reply apps/test/unit/lib/ui/SettingsCogTest.js]: Woops. Nope. Thanks for catching that!

File: apps/test/unit/lib/ui/SettingsCogTest.js (hunk 2 above)
Comment (epeach) [reply apps/test/unit/lib/ui/SettingsCogTest.js:102]: I follow your thinking for sure. An eyes or cucumber test does seem the best place for this type of check. Do you know if we have current coverage of this?

Comment (jmkulwik) [reply apps/test/unit/lib/ui/SettingsCogTest.js:102]: Talked offline and we decided that this isn't a necessary test.

**Review by epeach — APPROVED** [review]
LGTM!
